candid/governance.certified.idl.js

```diff
@@ -1,4 +1,4 @@
-/* Do not edit.  Compiled with ./scripts/compile-idl-js from candid/governance.did */
+/* Do not edit.  Compiled with ./scripts/compile-idl-js from candid//governance.did */
```

Comment:
The changes in here seem weird, there are more removed code than added, whereas in the candid file, there are more additions than removals.

And I found an example where the Record here does not match the candid file.

Could you check again?